src/goto-instrument/contracts/contracts.cpp

```diff
@@ -23,6 +23,7 @@ Date: February 2016
 #include <goto-instrument/havoc_utils.h>
 
 #include <goto-programs/goto_inline.h>
+#include <goto-programs/goto_program.h>
```

Comment:
I don't think this is strictly necessary? (Even the `remove_skip.h` will already drag in this header.)